app/controllers/spree/admin/orders_controller.rb

```diff
@@ -103,8 +103,8 @@ def print_ticket
         render template: "spree/admin/orders/ticket", layout: false
       end
 
-      def update_distribution_charge
-        @order.update_distribution_charge!
+      def recreate_all_fees
+        @order.recreate_all_fees!
```

Comment:
It would be great if we could make this private while we are at it. I see no reason why it should be public given it's not a controller action.

Reply:
I think it's already removed in a later commit in this PR :+1: